### PR TITLE
Added duo to the image

### PR DESCRIPTION
**Pull request by gal-forer**
## Status
Ready/In Progress/In Hold (Reason for hold)

## Related Content Pull Request
Related PR: link to the PR at demisto/content

## Related Issues
Related: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

### CODE REVIEW

**Review by jochman — CHANGES_REQUESTED** [review]
I might did not understood you.

Create a new docker image based on python3. Then add pydantic and duo.

I thought you already had a duo image and just wanted to add pydantic into it.

**Comment by gal-forer** [comment]
> I might did not understood you.
> 
> Create a new docker image based on python3. Then add pydantic and duo.
> 
> I thought you already had a duo image and just wanted to add pydantic into it.

I switched it after we talked

File: docker/duoadmin3/Pipfile [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+[dev-packages]
+
+[packages]
+fastapi = "*"
+uvicorn = "*"
+duo-client = "*"
+
+[requires]
+python_version = "3.8"
```
Comment (yaakovi) [inline docker/duoadmin3/Pipfile:14]: In the Dockerfile, you stated python 3.9.
Can we use 3.10?

**Review by yaakovi — APPROVED** [review]

File: docker/duoadmin3/Pipfile (hunk 1 above)
Comment (jochman) [reply docker/duoadmin3/Pipfile:14]: I'm encouraged to set it to python3.10.
pipenv locks to a specific python version; it is the best practice.

Comment (gal-forer) [reply docker/duoadmin3/Pipfile:14]: I need to check, the pr is not ready, I need to find the minimum I need to make it work, at the moment if I only add pydentic it doesn't work.
I added fastapi to the image and it did work, it's temporary for now so I'd be able to wok on duo for xsiam.
When everything will wirk with the integration I'll test the images

**Comment by gal-forer** [comment]
@yaakovi @jochman The pr is ready with the correct packages and it's python 3.10

**Review by jochman — APPROVED** [review]

**Pull request merged**